app/helpers/template_helpers.py

```diff
@@ -50,8 +50,10 @@ def get_footer_context(language_code, static_content_urls, sign_out_url, theme):
         "newTabWarning": lazy_gettext("The following links open in a new tab"),
         "copyrightDeclaration": {
             "copyright": lazy_gettext(
-                "Crown copyright and database rights 2020 OS 100019153."
-            ),
+                "Crown copyright and database rights 2021 NIMA MOU577.501."
+            )
+            if theme == "census-nisra"
+            else lazy_gettext("Crown copyright and database rights 2020 OS 100019153."),
```

Comment:
Are we 100% sure this shouldn't be 2021?

Reply:
I'll check with @donnap573

Reply:
Spoke to @donnap573 and in original email definitely says to put 2020

Reply:
🤷🏻‍♂️